src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -95,31 +116,33 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
     }
   }
 
-  const isDeleteDisabledFromApps = hasApps() && !_.isEmpty(nonDeletableApps)
-
   return h(Modal, {
     title: span({ style: { display: 'flex', alignItems: 'center' } }, [
       icon('warning-standard', { size: 24, color: colors.warning() }),
       span({ style: { marginLeft: '1ch' } }, ['Delete workspace'])
     ]),
     onDismiss,
     okButton: h(ButtonPrimary, {
-      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromApps,
+      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromResources,
       onClick: deleteWorkspace,
       tooltip: Utils.cond(
-        [isDeleteDisabledFromApps, () => 'You must ensure all apps in this workspace are deletable'],
+        [isDeleteDisabledFromResources && isGoogleWorkspace(workspace), () => 'You must ensure all apps in this workspace are deletable'],
+        [isDeleteDisabledFromResources && isAzureWorkspace(workspace), () => 'This workspace cannot be deleted'],
         [_.toLower(deleteConfirmation) !== 'delete workspace', () => 'You must type the confirmation message'],
-        () => 'Delete Workspace')
+        () => '')
     }, 'Delete workspace'),
     styles: { modal: { background: colors.warning(0.1) } }
   }, [
     div(['Are you sure you want to permanently delete the workspace ',
       span({ style: { fontWeight: 600, wordBreak: 'break-word' } }, name),
       '?']),
     getStorageDeletionMessage(),
-    hasApps() && div({ style: { marginTop: '1rem' } }, [
+    isDeleteDisabledFromResources && div({ style: { marginTop: '1rem' } }, [
```

Comment:
If delete is going to be disabled, it doesn't make sense to show "Deleting it will also delete any associated applications:".